<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserLoginAccount</class>
 <widget class="QWidget" name="UserLoginAccount">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>密码登录</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_account">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>291</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>登录（不可点击）</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_login_phone">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>291</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>短信验证码登录</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_protocol">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>311</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>我已阅读并同意《软件协议》《个人信息保护指引》</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_wechat">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>微信</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_qq">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>qq</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_help">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_look_password">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>显示密码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
